tests/unit/mixins/test_seriliazable_mixin.py

```diff
@@ -0,0 +1,3 @@
+import pytest
```

Comment:
did you intend to include this file in this PR?

Reply:
I did not!